js/app.js

```diff
@@ -150,15 +150,17 @@ const ctx = board.getContext("2d");
 ctx.globalCompositeOperation = 'destination-over';
 
 /* Export/Download canvas image. */
-const downland = () => {
+const downland = (withBg = false) => {
     const link = document.createElement('a');
     const locale = new Date().toLocaleString();
     const filename = `board${locale}.png`;
     link.download = filename;
 
-    ctx.globalCompositeOperation="destination-over";
-    ctx.fillStyle = localStorage.getItem("bcolor");
-    ctx.fillRect(0, 0, ctx.canvas.width, ctx.canvas.height);
+    if (withBg) {
```

Comment:
you should remove this if condition and pass the statements to `if (type == 'download-with-bg') {` here
and remove param withParam becausse it didnt need at all,